.gitignore

```diff
@@ -9,4 +9,4 @@
 /cc65.zip
 /util/atari/*.exe
 /util/gamate/*.exe
-
+targettest/cbm/cbmread.prg
```

Comment:
would you remove this from the PR please?

Reply:
Why? What's wrong with it?

Reply:
Well. IF we start adding the test binaries to the git ignores... then all of them, using proper wildcards. a single one is just weird :) (and i prefer different things in separate commits)

Reply:
Sounds a bit like bike shedding looking at the current file contents, but ok.